app/core/lib/slash_commands/ticket_dependency.py

```diff
@@ -94,19 +93,14 @@ def command_ticket_dependency(self, match) -> str:
                             to_ticket = self.ticket.__class__.objects.get(pk = ticket_id)
 
 
-                        item = ModelSerializer(
-                            data = {
-                                'ticket': from_ticket.id,
-                                'how_related': int(how_related),
-                                'dependent_ticket': to_ticket.id,
-                                'organization': from_ticket.organization.id
-                            }
-                        )
-
-                        if item.is_valid( raise_exception = False ):
-
-                            item.save()
+                        TicketDependency.objects.create(
+                            ticket = from_ticket,
+                            how_related = how_related,
+                            dependent_ticket = to_ticket,
+                            organization = from_ticket.organization,
+                            user = self.user
```

Comment:
currently this'll only work for ticket comment. if a ticket dependency is added via ticket description, either update/create. This will fail to fetch the user